editor.planx.uk/src/pages/Preview/Context.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import type { Flow, GlobalSettings } from "../../types";
+import type { FlowSettings, GlobalSettings } from "../../types";
 
 export const PreviewContext = React.createContext<
```

Comment:
I think we should remove this.

We're currently using three methods of state management -

- React Context (here)
 - Zustand stores
- Route data (using withData() in routes)

I'd be in favour of simplifying this down to just Zustand. There's a few places here that this would simplify things but I'd not touched them in order to keep this PR as focussed as possible.